x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/rest/RestClearSearchableSnapshotsCacheAction.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.searchablesnapshots.rest;
+
+import org.elasticsearch.action.support.IndicesOptions;
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.rest.BaseRestHandler;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+import org.elasticsearch.xpack.searchablesnapshots.action.ClearSearchableSnapshotsCacheAction;
+import org.elasticsearch.xpack.searchablesnapshots.action.ClearSearchableSnapshotsCacheRequest;
+
+import java.util.List;
+
+public class RestClearSearchableSnapshotsCacheAction extends BaseRestHandler {
+
+    @Override
+    public List<Route> routes() {
+        return List.of(
+            new Route(RestRequest.Method.POST, "/_searchable_snapshots/cache/clear"),
+            new Route(RestRequest.Method.POST, "/{index}/_searchable_snapshots/cache/clear")
+        );
+    }
+
+    @Override
+    public String getName() {
+        return "clear_searchable_snapshots_indices_action";
```

Comment:
I think we should have `cache` in the  name here somewhere. I tried a couple of different word orders and none was perfect. Maybe this?

```suggestion
        return "clear_indices_searchable_snapshots_cache_action";
```

Reply:
Thanks for spotting this. I pushed f1ce506f4605ebf0fe669db0a74e6c2e61fc765a to use your suggestion.